src/routes/index.svelte.md

```diff
@@ -101,6 +102,10 @@ blocked again. This may have been the cause for the day's earlier outage.
 
 While these incidents were in no way intentional, it highlights the poor QA testing that Manjaro performs. This has happened on two separate occasions in less than two years.
 
+## Miscellaneous
+
+[Their system update script runs `rm` on the lockfile mid-transaction.](https://gitlab.manjaro.org/packages/core/manjaro-system/blob/3b806753e245b7ec7e18bb674e916e28d751a429/manjaro-update-system.sh#L45(https://archive.fo/dofw8)) The lockfile is in place to prevent multiple instances of pacman from trying to alter the package database at the same time. Sometimes, when pacman is interrupted the stale lockfile can remain, in those cases removing the lockfile is a common troubleshooting step. However you should only do that when you are **absolutely certain** there are no other pacman instances running and Manjaro's script does this preventively, without even prompting the user.
```

Comment:
i've done a little more digging and it seems like this is an old issue that was fixed a while ago (it's no longer [in current versions](https://gitlab.manjaro.org/packages/core/manjaro-system/-/blob/master/manjaro-update-system.sh))

i dont know if this should be left out, or include it with the context of "they've fucked this up in the past"